src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java

```diff
@@ -219,7 +237,50 @@ public String getTaskDefinitionArn() {
         return items;
     }
 
-    public static class EnvironmentEntry {
+    public List<MountPointEntry> getMountPoints() {
+        return mountPoints;
+    }
+
+    public List<VolumeEntry> getVolumes() {
+        return volumes;
+    }
+
+    private Collection<Volume> getVolumeVolumes() {
```

Comment:
Method name is a bit surprising, I suggest `getVolumeSpec` or anything that make it clean what type of Volume we are talking about here